docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -42,9 +55,40 @@ Data Description
 
 State and Transition Data Tables
 ++++++++++++++++++++++++++++++++
-   
-Validation Criteria
--------------------
 
-References
-----------
+.. list-table:: State Data
+   :widths: 5 10 10 20
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - MAM
+     - disability weight
+     - :math:`\frac{{\sum_{sequelae\in \text{MAM}}} \scriptstyle{\text{disability_weight}_s \times\ \text{prevalence}_s}}{{\sum_{sequelae\in \text{MAM}} \scriptstyle{\text{prevalence}_s}}}`
+     - disability weight for MAM
+   * - SAM
+     - disability weight
+     - :math:`\frac{{\sum_{sequelae\in \text{SAM}}} \scriptstyle{\text{disability_weight}_s \times\ \text{prevalence}_s}}{{\sum_{sequelae\in \text{SAM}} \scriptstyle{\text{prevalence}_s}}}`
+     - disability weight for SAM
+
+
+.. list-table:: Data Sources and Definitions
+   :widths: 10 10 20 20
+   :header-rows: 1
+
+   * - Variable
+     - Source
+     - Description
+     - Notes
+   * - MAM sequelae
+     - 
+     - {s198, s2033}
+     - Moderate wasting with edema, moderate wasting without edema
```

Comment:
Maybe we can spell edema with an 'o' to be consistent with how GBD spells it in the risk appendix?